shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -281,12 +281,63 @@ void AccessibilityBridge::UpdateScreenRects(
   }
 }
 
+std::optional<flutter::SemanticsAction>
+AccessibilityBridge::GetFlutterSemanticsAction(
+    fuchsia::accessibility::semantics::Action fuchsia_action,
+    uint32_t node_id) {
+  switch (fuchsia_action) {
+    /// The default action associated with the element.
+    case fuchsia::accessibility::semantics::Action::DEFAULT:
+      return flutter::SemanticsAction::kTap;
+    /// The secondary action associated with the element. This may correspond to
+    /// a long press (touchscreens) or right click (mouse).
+    case fuchsia::accessibility::semantics::Action::SECONDARY:
+      return flutter::SemanticsAction::kLongPress;
+    /// Set (input/non-accessibility) focus on this element.
+    case fuchsia::accessibility::semantics::Action::SET_FOCUS:
+      FML_DLOG(WARNING)
```

Comment:
Can this just go in the default case where all unsupported actions are logged? Not sure why this is a warning but the others are an assertion (which wont be visible in opt builds).

Reply:
I can switch the default case, but I think it's helpful to have a friendly string printed out rather than an integer ID.  WDYT?

Reply:
I guess part of the idea was that if Fuchsia adds more actions on us we'll want to assert.  But it's probably better not to crash in that case.